launchdarkly/provider_test.go

```diff
@@ -5,9 +5,8 @@ import (
 	"os"
 	"testing"
 
-	"github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
-	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
```

Comment:
I'm seeing some errors in this file
```
launchdarkly/provider_test.go:14:33: undefined: terraform.ResourceProvider
launchdarkly/provider_test.go:18:30: invalid type assertion: Provider().(*schema.Provider) (non-interface type *schema.Provider on left)
```
According to [this](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#removal-of-the-terraform-resourceprovider-interface) we can replace `terraform.ResourceProvider` with `*schema.Provider` and remove the type assertion on line 18.